### PR TITLE
fix broken link docs

### PR DESCRIPTION
**Pull request by laisbsc**
## Description
Draft PR to fix some broken links I have been finding on the docs.

## Development notes
Changed the broken links and checked them manually before pushing the commit.

## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) guidelines
- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change and added my name to the list of supporting contributions in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/master/RELEASE.md) file
- [ ] Added tests to cover my changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: docs/source/02_get_started/05_example_project.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ For project-specific settings to share across different installations (for examp
 
 The folder contains three files for the example, but you can add others as you require:
 
--   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
+-   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog.html#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
```
Comment (deepyaman) [inline docs/source/02_get_started/05_example_project.md:58]:
```suggestion
-   `catalog.yml` - [Configures the Data Catalog](../05_data/01_data_catalog.md#using-the-data-catalog-within-kedro-configuration) with the file paths and load/save configuration required for different datasets
```
Pretty sure it should point to the Markdown file, and the docs will get generated correctly. See https://github.com/quantumblacklabs/kedro/blame/72ba895d3db8ecd2efae3419b298877138145291/docs/source/05_data/01_data_catalog.md#L266 for an example.

Comment (laisbsc) [reply docs/source/02_get_started/05_example_project.md:58]: I think you're right. There was just no extension there and the link was returning a 404.
Thanks for the correction, though.

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Review by 921kiyo — APPROVED** [review]
Thanks! Meanwhile, could you also fix these two broken links too?

Screenshot 2020-10-02 at 14 47 24

**Pull request merged**